config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider:              'AWS',
+    aws_access_key_id:     Rails.application.secrets.amazon_key,
+    aws_secret_access_key: Rails.application.secrets.amazon_secret,
+  }
+  config.fog_directory  = 'flashcards-storage'                         # required
```

Comment:
Line is too long. [81/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.